src/common/Icons.js

```diff
@@ -28,46 +30,62 @@ type IconProps<Glyphs: string> = {|
   color?: Color,
 |};
 
+/** A variation of `IconProps` closer to what upstream's .js.flow says. */
+type IconPropsBusted<Glyphs: string> = {| ...IconProps<Glyphs>, allowFontScaling: boolean |};
```

Comment:
Why not just `import { IconProps as IconPropsBusted } from 'react-native-vector-icons'`?

Or alternatively, instead `import { Icon as IconBusted } from 'react-native-vector-icons'`, and use `$Class<IconBusted<Glyphs>>` (I think) in place of `ComponentType<IconPropsBusted<Glyphs>>` below.

Reply:
> Why not just `import { IconProps as IconPropsBusted } from 'react-native-vector-icons'`?

Hmm, indeed that seems to work! I'll do that.

I thought I'd tried something like that and it didn't, but something must have been different.


> Or alternatively, instead `import { Icon as IconBusted } from 'react-native-vector-icons'`, and use `$Class<IconBusted<Glyphs>>` (I think) in place of `ComponentType<IconPropsBusted<Glyphs>>` below.

It feels like something on these lines ought to work too, but if I try it (with [`Class`](https://flow.org/en/docs/types/utilities/#toc-class)) I get a bunch of Flow errors like this:
```
Cannot call makeIcon with Feather bound to iconSet because Icon [1] is
incompatible with Icon [2].

     src/common/Icons.js
 [2] 54│   iconSet: Class<IconBusted<Glyphs>>,
       :
     90│ export const IconPlusSquare = makeIcon(Feather, 'plus-square');

     node_modules/react-native-vector-icons/Feather.js.flow
 [1]  9│ declare export default Class<Icon<FeatherGlyphs>>;
```

Nominal typing? Something else? Dunno.

Possibly this is the thing I tried that I remembered as also being `IconProps as IconPropsBusted`.